src/generators/generator.js

```diff
@@ -17,7 +17,7 @@ class Generator {
     this.componentName    = args.componentName;
     this.templatePath     = args.templatePath;
     this.testTemplatePath = args.testTemplatePath;
-
+    this.wrapFilesInFolders = args.wrapFilesInFolders;
     // project wide settings
     this.sourceBase    = args.settings.getSetting('sourceBase');
     this.testBase      = args.settings.getSetting('testBase');
```

Comment:
Hmm yeah thats a tough call I totally see what you mean though.  I feel like it makes sense to err on the side of all or nothing/I'm wary of letting people configure stuff TOO much.  There are a lot of advantages of just having certain hard conventions.  Wish someone could like poll all React developers to get most used conventions and then just use those haha